application/src/main/resources/config/application.yaml

```diff
@@ -85,4 +85,22 @@ ledger-sync:
     pool-offline-data:
       fixed-delay: ${POOL_OFFLINE_DATA_FIXED_DELAY:172800}
       initial-delay: ${POOL_OFFLINE_DATA_INIT_DELAY:20000}
-    enabled: ${SCHEDULER_ENABLED:true}
\ No newline at end of file
+    enabled: ${SCHEDULER_ENABLED:true}
+  healthcheck:
+    block-time-threshold: ${BLOCK_TIME_THRESHOLD_IN_SECOND:240}
+    inserted-time-threshold: ${INSERTED_TIME_THRESHOLD_IN_SECOND:1200}
```

Comment:
Isn't that too long? 20 minutes

Reply:
This threshold is used for both cases: batching and non-batching, so I set the default value to be larger than normal.
We could use two separate thresholds for each case. However, the drawback is that we have to add environment variables, which could introduce unnecessary complexity.